app/models/faq.py

```diff
@@ -9,11 +9,13 @@ class Faq(db.Model):
     answer = db.Column(db.String, nullable=False)
     event_id = db.Column(db.Integer,
                          db.ForeignKey('events.id', ondelete='CASCADE'))
+    faq_type_id = db.Column(db.Integer, db.ForeignKey('faq_types.id', ondelete='CASCADE'))
```

Comment:
Isn't FAQ to FAQ-Type many-to-many relation? How are you gonna store multiple faq_type_id in a single integer field? Maybe you can use comma-separate-integers string.

Reply:
No, FAQ to FAQ-type is a many to one relation. A single FAQ can only belong to one Type